src/main/java/org/jabref/logic/shared/DBMSProcessor.java

```diff
@@ -401,16 +401,22 @@ private void insertOrUpdateFields(BibEntry localBibEntry) throws SQLException {
      *
      * @param bibEntry {@link BibEntry} to be deleted
      */
-    public void removeEntry(BibEntry bibEntry) {
+    public void removeEntries(List<BibEntry> bibEntries) {
         StringBuilder query = new StringBuilder()
                 .append("DELETE FROM ")
                 .append(escape("ENTRY"))
                 .append(" WHERE ")
                 .append(escape("SHARED_ID"))
-                .append(" = ?");
+                .append(" IN (");
+        for (int i = 0; i < bibEntries.size() - 1; i++) {
+            query.append("?, ");
```

Comment:
I think you can use `"?, "`.repeat(bibEntries.size())` here

Reply:
Made the change, will push soon.